http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWdsDAOSpec.scala

```diff
@@ -46,28 +46,56 @@ class HttpWdsDAOSpec extends AnyFlatSpec with Matchers with LeonardoTestSuite wi
     )
 
     val wdsDAO = new HttpWdsDAO(okWds)
-    val res = wdsDAO.getStatus(Uri.unsafeFromString("https://test.com/wds/status"), authHeader)
+    val res = wdsDAO.getStatus(Uri.unsafeFromString("https://test.com/wds/status"), authHeader, AppType.Cromwell)
     val status = res.unsafeRunSync()
     status shouldBe false
   }
 
-  "HttpWdsDAO.getStatus" should "return true if status is ok" in {
+  "HttpWdsDAO.getStatus" should "return true if status is ok for both cromwell and WDS appTypes" in {
     val authHeader = Authorization(Credentials.Token(AuthScheme.Bearer, "token"))
 
-    val response =
+    val upResponse =
+      """
+        |{
+        |  "status": "DOWN"
+        |}
+      """.stripMargin
+
+    val downResponse =
```

Comment:
nit: looks like the `upResponse` and `downResponse` are transposed - I don't think this has effect on the tests but could be confusing variable naming.

Reply:
ah! the test was broken. we were returning "DOWN" when "UP" was expected, and the flipped naming here made it incorrectly pass! I added positive and negative cases to the test so it would fail in the future if names were flipped. Fixed now :)